examples/audio/main/bsp_audio_example.c

```diff
@@ -10,40 +10,30 @@
 #include <unistd.h>
 #include <string.h>
 #include <inttypes.h>
+#include <sdkconfig.h>
 #include "esp_log.h"
 #include "esp_spiffs.h"
```

Comment:
Remove all includes except:
```
#include <sdkconfig.h>
#include "esp_log.h"
#include "bsp/esp-bsp.h"
```

Reply:
Thank you for the comment! I removed some, but not all:
I left `inttypes.h` in there because we use PRix macrose in this file and I left `stdio.h` because we use file-handling functions in this file.

I also added freertos/task.h and freertos/queue.h as they were missing in the includes list